meetings/2018-07-12.md

```diff
@@ -0,0 +1,74 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-07-11
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=jX_FFG-XC_E
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/565
+
+## Present
+
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* James M Snell @jasnell (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Timothy Gu @TimothyGu (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/build
+
+* Request for elevated permissions [#1337](https://github.com/nodejs/build/issues/1337)
+  * Rich setting up meeting, leave to build WG to progress for now.
```

Comment:
Uh....nope, not me.  I think @joyeecheung tried but it didn't work out?  Maybe @rvagg is addressing this? Would it be frustrating to kick this back to the Build WG to work out and escalate back to TSC only if that doesn't yield a satisfying outcome?

Reply:
Meh, never mind, I'll take on the task of figuring out a meeting (unless someone else volunteers) just to keep this moving forward.